remix/app/components/FormInput/error.tsx

```diff
@@ -0,0 +1,14 @@
+interface ErrorProps {
+  error?: string;
+}
+
+export default function Error({ error }: ErrorProps) {
+  return (
+    <p
```

Comment:
Semantically-wise, it might be better to use `div` or `small` tag as `p` tags requires lots of style overrides such as margins

Reply:
Yep will prove that.

Reply:
Improve here: [0af22fb](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/0af22fb487b5618b545e2edf90f9b9e0fe94da67)